peps/pep-0696.rst

```diff
@@ -137,9 +137,16 @@ should flag this an error.
        AllTheDefaults[int, complex, str, int, bool]
    )  # All valid
 
-This cannot be enforced at runtime for functions, for now, but in the
-future, this might be possible (see `Interaction with PEP
-695 <#interaction-with-pep-695>`__).
+With the new Python 3.12 syntax for generics (introduced by :pep:`695`), this can
+be enforced at compile time:
+
+.. code-block:: py
```

Comment:
Tip: Python is the default, all these `.. code-block:: py` (and one `python`) could be replaced with the double colon. But not a big deal.

Reply:
Good point, removed those and fixed a few other formatting nits